<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="8,7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="lbl_fazer_hoje">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>19</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Fazer hoje</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_teste">
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Duplo clique para cadastrar progressos</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tb_fazer_hoje">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="rowCount">
           <number>3</number>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <row/>
          <row/>
          <row/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,9">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>19</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Metas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_status">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>Em andamento</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Concluidas</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Canceladas</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="list_metas">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="movement">
           <enum>QListView::Static</enum>
          </property>
          <property name="viewMode">
           <enum>QListView::ListMode</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="8,3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>19</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Fazer essa semana</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tb_fazer_essa_semana">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="rowCount">
           <number>3</number>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <row/>
          <row/>
          <row/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,1">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="btn_cadastrar_meta">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Cadastrar meta</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_calendario">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Calendario</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_sair">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Sair</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
